aiidalab/config.py

```diff
@@ -18,11 +19,11 @@
     click.secho("\n".join([f"\U0001F6A7  {line}" for line in lines]), fg="yellow")
 
 
-def _as_env_var_name(key):
+def _as_env_var_name(key: str) -> str:
     return "AIIDALAB_" + key.upper()
 
 
-def _get_config_value(key, default=None):
+def _get_config_value(key: str, default: str) -> Any:
```

Comment:
```suggestion
def _get_config_value(key: str, default: str | None) -> Any:
```

`default` is passed to `os.getenv` and accept to be `None`.